### PR TITLE
Adding mandatory resource tracking to Managed App template

### PR DESCRIPTION
**Pull request by EugeneShashkevich**
Changes:
-template and documentation changes

### CODE REVIEW

**Comment by linclim** [comment]
we shouldn't put in any hardcoded/dummy value for the resourcing tracking guid in our sample template.

**Comment by EugeneShashkevich** [comment]
>           we shouldn't put in any hardcoded/dummy value for the resourcing tracking guid in our sample template.

done

File: ManagedApp/mainTemplate.json [hunk 1]
```diff
@@ -156,6 +156,19 @@
       ],
       "location": "[resourceGroup().location]",
       "resources": [
+		{ 
+			"apiVersion": "2018-02-01",
+			"name": "pid-XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX", // use your generated GUID here for resource tracking as described at https://aka.ms/aboutinfluencedrevenuetracking and remove these comments
```
Comment (krthk) [inline ManagedApp/mainTemplate.json]: Change format to be consistent with the convention in the rest of the file - 

"name": "<valid resource tracking guid>"

Comment (EugeneShashkevich) [reply ManagedApp/mainTemplate.json]: the format is taken from their docs so if they read that URL, did not want customers to see anything different vs related documentation. Also the format is: pid-<guid> not a <guid>

Comment (EugeneShashkevich) [reply ManagedApp/mainTemplate.json]: Merged both version, I think it is consistent now and also gives necessary link

**Pull request merged**